CONTRIBUTING.md

```diff
@@ -11,13 +11,14 @@ We strongly recommend that you set up these git hooks on your machine by:
 ```sh
 # sh
 # in the project root, run:
-ln -sf ../../.scripts/pre-commit .git/hooks/pre-commit
```

Comment:
This is not a typo, pls see this answer on [SO](https://stackoverflow.com/questions/4592838/symbolic-link-to-a-hook-in-git#4594681).
I can confirm that the instructions in this doc work perfectly for me.
That said, I just discovered [this super cool dev tool](https://github.com/qgadrian/elixir_git_hooks) to simplify sharing of git-hooks. On the outset, it looks like a godsend but it is a bit... rude.
It installs git-hooks when you do `mix deps.get` (after backing up existing hooks -- almost silently). Most people (including me) will get furious if a project _forces_ me to _develop code_ in someone else's (opinionated) way.

Reply:
Yeah! Got the point. It uses the path relative to `.git/hooks`.  I think it makes sense to add the brief explanation. It looks confusing why should one move back two levels of hierarchy.

Thank you for clarifying.